tests/Integration/Behaviour/Features/Scenario/Carrier/carrier_management.feature

```diff
@@ -8,20 +8,34 @@ Feature: Carrier management
   As a BO user
   I must be able to create, edit and delete carriers
 
+  Background:
+    Given group "visitor" named "Visitor" exists
+    Given group "guest" named "Guest" exists
+
   Scenario: Adding new Carrier
     When I create carrier "carrier1" with specified properties:
       | name         | Carrier 1                          |
       | grade        | 1                                  |
       | trackingUrl  | http://example.com/track.php?num=@ |
       | position     | 2                                  |
       | active       | true                               |
+      | max_width    | 1234                               |
+      | max_height   | 1234                               |
+      | max_depth    | 1234                               |
+      | max_weight   | 1234                               |
```

Comment:
Maybe use different values, just to make sure each field is correctly handled independently